src/utils/additional-headers.ts

```diff
@@ -22,6 +23,10 @@ export const getAdditionalHeaders = ({
     header['x-tt-anonymous-id'] = anonymousId
   }
 
+  if (affiliateId) {
+    header['affiliateId'] = affiliateId
```

Comment:
this header already exists on the service under this key, that's why is not being used x-tt- preffix